### ISSUE TITLE
node-specificity explore - TypeError: Cannot call method 'toFixed' of null

### ISSUE THREAD

**Issue by victor-homyakov** [eprev/specificity#14]
```
C:\temp>node-specificity explore css-specificity.json >css-specificity.txt

C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\src\reports\inspect.js:14
        return parseInt(v) === v ? v : v.toFixed(2);
                                         ^
TypeError: Cannot call method 'toFixed' of null
    at toFixed (C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\src\reports\inspect.js:14:42)
    at Array.map (native)
    at toFixed (C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\src\reports\inspect.js:12:18)
    at C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\src\reports\inspect.js:73:17
    at Array.forEach (native)
    at printSummary (C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\src\reports\inspect.js:57:27)
    at C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\src\reports\inspect.js:101:13
    at Array.forEach (native)
    at module.exports (C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\src\reports\inspect.js:83:10)
    at Command.<anonymous> (C:\Users\xxx\AppData\Roaming\npm\node_modules\node-specificity\bin\specificity:134:13)
```

**Comment by victor-homyakov** [comment]
The output file is created, but contains only the first table (list of selectors). Second table (max/avg/med specificity) and histogram are missing.

**Comment by eprev** [comment]
Victor, could you please share `css-specificity.json` (gist or pastebin, for instance, will work)?

**Comment by victor-homyakov** [comment]
https://gist.github.com/victor-homyakov/957c1f4045754b2c4fd3 (2.3M in size)

**Comment by victor-homyakov** [comment]
Reproduced on minimal test case: CSS with `@import` only:

```CSS
@import url('partial/animation.css');
@import url('partial/typography.css');
```

**Comment by victor-homyakov** [comment]
Parsed JSON looks like

```json
{"files":{"*":{"series":[],"important":{"min":null,"max":0,"avg":null,"med":0},"weight_a":{"min":null,"max":0,"avg":null,"med":null},"weight_b":{"min":null,"max":0,"avg":null,"med":null},"weight_c":{"min":null,"max":0,"avg":null,"med":null},"weight":{"min":[null,null,null],"max":[0,0,0],"avg":[null,null,null],"med":null},"distrib":{}}},"options":{"cwd":"C:\\temp","uniqueSelectors":false,"label":""}}
```

and inspect.js tries to execute `[null, null, null].map(toFixed)`

**Comment by eprev** [comment]
Thanks for your report. This have been fixed in v0.1.11.
